src/darsia/utils/linear_solvers/jacobi.py

```diff
@@ -0,0 +1,135 @@
+"""
+Jacobi solver for linear systems. To be used either as a solver or as a smoother.
+"""
+from __future__ import annotations
+
+from typing import Optional, Union
+
+import numpy as np
+
+import darsia as da
+
+
+class Jacobi(da.Solver):
+    """
+    Jacobi solver for the problem: mass_coeff * x - diffusion_coeff * laplace(x) = rhs
+    """
+
+    def __init__(
+        self,
+        mass_coeff: Union[float, np.ndarray],
```

Comment:
Just to note, did you test this (and the other solvers) for array inputs? I cannot see any use of np.multiply or np.divide, which I would have expected instead of "/" for the latter.

Reply:
I tested it all for array inputs, and im like 99% sure that np.multiply is equivalent with * (similar with / and divide)

Reply:
It might be that I have misunderstood though..?

Reply:
I think you should be right. Disregard my comment.